nrf_modem/doc/CHANGELOG.rst

```diff
@@ -12,14 +12,14 @@ All notable changes to this project are documented in this file.
 nrf_modem 1.0.2
 ***************
 
-* Implement RAI (Release Assistance Indication) support in modem lib.
-* Fixed an issue that would lead to both POLLIN and POLLHUP being reported by poll() when a connection is closed by the peer.
-* Fixed non-blocking recv behavior when WAITALL or DONTWAIT flags was used.
-* Fixed an issue where a blocking send could in some cases return before all data was sent.
-* Reduced Heap memory usage in recv() by 20% when using IPv4.
+* Implemented RAI (Release Assistance Indication) support in Modem library.
+* Fixed an issue that leads to the reporting of both POLLIN and POLLHUP by poll() when a connection is closed by the peer.
+* Fixed the non-blocking recv() behavior when WAITALL flag or DONTWAIT flag was used.
```

Comment:
I think we need to rephrase this line:

Fixed an issue where a recv call on a non-blocking socket would not always behave correctly when the MSG_WAITALL or MSG_DONTFLAGS flag was used.

Reply:
Rephrased.